### DISCUSSION TITLE
Permission error message help please

### DISCUSSION THREAD

**Discussion by Mirandp** [community/community#22006] [New to GitHub]
<p>warning: unable to access ‘/Users/me/.config/git/ignore’: Permission denied<br>
warning: unable to access ‘/Users/me/.config/git/attributes’: Permission denied<br>
warning: unable to access ‘/Users/me/.config/git/attributes’: Permission denied<br>
warning: unable to access ‘/Users/me/.config/git/ignore’: Permission denied<br>
On branch gh-pages<br>
Your branch is up to date with ‘origin/gh-pages’.</p>

**Comment by yamiacat** [comment]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>This looks like a permissions error on your local machine. Your user account doesn’t have permission to access some of the files Git needs.</p>
<p>This stack overflow answer should point you in the right direction!</p>
<aside class="onebox stackexchange">
  <header class="source">
      <a href="https://stackoverflow.com/questions/44815139/config-git-ignore-permission-denied" target="_blank">stackoverflow.com</a>
  </header>
  <article class="onebox-body">
      <a href="https://stackoverflow.com/users/8128359/dennis-digital" target="_blank">
    <img alt="Dennis Digital" src="https://user-images.githubusercontent.com/19803303/181068207-20204058-9b48-4522-8a5d-71ba96ffc8e5.png" class="thumbnail onebox-avatar" width="128" height="128">
  </a>
<h4>
  <a href="https://stackoverflow.com/questions/44815139/config-git-ignore-permission-denied" target="_blank">.config/git/ignore': Permission denied</a>
</h4>

<div class="tags">
  <strong>git</strong>
</div>

<div class="date">
  asked by
  
  <a href="https://stackoverflow.com/users/8128359/dennis-digital" target="_blank">
    Dennis Digital
  </a>
  on <a href="https://stackoverflow.com/questions/44815139/config-git-ignore-permission-denied" target="_blank">01:54AM - 29 Jun 17 UTC</a>
</div>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by Mirandp** [comment]
<p>Hello!</p>
<p>I tried this but it unfortunately isn’t solving the problem. I’m still getting this error from git status:</p>
<p>My-MacBook-Pro: class-test myusername$ git status<br>
warning: unable to access ‘/Users/myusername/.config/git/ignore’: Permission denied<br>
On branch gh-pages<br>
Your branch is up to date with ‘origin/gh-pages’.</p>
<p>Thank you!</p>

**Comment by Mirandp** [comment, marked as answer]
<p>I used this command:</p>
<p>sudo chown -R <span class="math">USER:</span>(id -gn $USER) /Users/myusername/.config</p>
<p>and I think that fixed the problem.</p>

**Comment by yamiacat** [comment]
<p>I’m glad you got it working!</p>

**Comment by undavide** [comment]
<p>With <code>zsh</code> (standard shell on newer macOS) the command exits with an <code>unknown file attribute: i</code> error.<br>
Could you please provide a <code>zsh</code>-safe version of it? Thanks!</p>

**Comment by yamiacat** [comment]
<p>Hi <a class="mention" href="/undavide">@undavide</a> ! :wave: Welcome to the Community!</p>
<p>I thiiiink that error is caused by zsh failing to use a variable, so you should be able to run the commands separately rather than together.</p>
<p>First, find your username with:</p>
<pre data-code-wrap="zsh"><code class="lang-nohighlight">id -un
</code></pre>
<p>…then use that value to run:</p>
<pre data-code-wrap="zsh"><code class="lang-nohighlight">sudo chown -R USERNAME /Users/USERNAME/.config
</code></pre>

**Reply by noob339** [reply]
You're a godsend, thank you, that worked!

**Reply by saraoros** [reply]
This worked for me as well! Thank you so much!

**Reply by Karogh** [reply]
Thanks.

**Reply by comet7001** [reply]
Thanks a lot😂

**Reply by uurdmc** [reply]
Thank you! It works!

**Reply by gaussfff** [reply]
Thx! It's working!

**Reply by Manzimcfly** [reply]
hi! I try with this command but I got, chown: USERNAME: illegal user name :( any suggestion... thank you!

**Reply by rvalenzuelar** [reply]
Thanks!

**Reply by bharadwaj3000** [reply]
Thanks a lot
